CHANGELOG.md

```diff
@@ -10,12 +10,17 @@ Description of the upcoming release here.
 
 ### Added
 
-- [#1274](https://github.com/FuelLabs/fuel-core/pull/1274): Added tests to benchmark block synchronization.
+- [#1322](https://github.com/FuelLabs/fuel-core/pull/1322): 
+    The `debug` flag is added to the CLI. The flag should be used for local development only. Enabling debug mode:
+    - Allows GraphQL Endpoints to arbitrarily advanced blocks.
+    - Enabled debugger GraphQL Endpoints.
+    - Allows setting `utxo_validation` to `false`.
```

Comment:
This may be a breaking change for the sway quickstart guides / lsp tooling that automatically spins up fuel-core. I think it makes sense to protect this setting, but just calling it out that this is a breaking change that needs to be flagged across all the tooling and docs that use fuel-core.

Reply:
This will also be somewhat difficult to trace down all occurrences, since searching across the repos for `--utxo-validation` will only show the places that already do use proper validation. But there are likely many places that describe how to startup nodes that omit this flag.

Reply:
If this flag is omitted, then it is `true` by default, and we don't need to worry=)